documentation/getting-started/README.md

```diff
@@ -5,44 +5,51 @@ permalink: /documentation/getting-started/
 ---
 # Getting Started
 
-This Getting Started tutorial describes how to record a test, run it and analyze the results.
+This Getting Started tutorial describes how to record a test for the [TestCafe Studio sample page](http://devexpress.github.io/testcafe/example), run the test and analyze the results.
 
-This tutorial uses the [TestCafe Studio sample page](http://devexpress.github.io/testcafe/example) to record the test and consists of the following sections:
+The tutorial consists of the following steps:
 
 * [Install TestCafe Studio](#install-testcafe-studio)
-* [Record Test Actions](#record-test-actions)
+* [Start Recording a Test](#start-recording-a-test)
+* [Interact with the Tested Webpage](#interact-with-the-tested-webpage)
 * [View the Recorded Test](#view-the-recorded-test)
-* [Add New Actions to the Test](#add-new-actions-to-the-test)
+* [Add an Assertion to the Test](#add-an-assertion-to-the-test)
 * [Run the Test](#run-the-test)
 * [View the Test Results](#view-the-test-results)
 
 ## Install TestCafe Studio
 
 You can install TestCafe Studio on MacOS, Windows or Linux. See [Installation](../guides/installation.md) for more information.
 
-## Record Test Actions
+## Start Recording a Test
 
-First, create a directory to store your tests.
-Go to TestCafe Studio, open the **File** menu, click **Open Tests Directory** and navigate to the created directory.
-You can view the directory's contents in the [Explorer](../user-interface/explorer-panel.md) panel.
+When you open TestCafe Stuido at the first time, it shows the **Welcome** page that allows you to start recording a test quickly. Enter the tested site's URL ([http://devexpress.github.io/testcafe/example](http://devexpress.github.io/testcafe/example)) and click the ![Record button](../../images/getting-started/record-test-icon.png) **Start recording a test** button. TestCafe Studio creates a test, launches a browser with the tested site and starts recording your activity on the webpage (see [Interact with the Tested Page](#interact-with-the-tested-page)).
 
-![The Explorer panel](../../images/getting-started/explorer-panel.png)
+![The Welcome page](../../images/getting-started/welcome-page.png)
 
-Next, create a test fixture for the sample webpage tests.
-In the **Explorer** panel, click the **Create a New Fixture** button, and in the invoked **New Fixture** dialog, specify the fixture name and the tested page's URL. Click **Create**.
+You can also organize your tests and start the recording from the [Explorer](../user-interface/explorer-panel.md) panel. Follow the steps:
```

Comment:
'Follow the steps:' Follow which steps? and follow these steps to do what?

Reply:
To do this _(organize the test and start recording)_, follow the steps below: